tests/PHPUnit/Integration/ReleaseCheckListTest.php

```diff
@@ -617,6 +616,47 @@ public function test_TotalPiwikFilesSize_isWithinReasonnableSize()
         $this->assertGreaterThan($minimumTotalFilesizesExpectedInMb * 1024 * 1024, $sumFilesizes, "expected to have at least $minimumTotalFilesizesExpectedInMb Mb of files in Piwik codebase.");
     }
 
+    public function test_noUpdatesInCorePlugins()
+    {
+        $manager = Manager::getInstance();
+        $plugins = $manager->loadAllPluginsAndGetTheirInfo();
+
+        $pluginsWithUnexpectedUpdates = array();
+        $pluginsWithUpdates = array();
+        $numTestedCorePlugins = 0;
+        foreach ($plugins as $pluginName => $info) {
+            if ($manager->isPluginBundledWithCore($pluginName)) {
+                $numTestedCorePlugins++;
+                $pathToUpdates = Manager::getPluginDirectory($pluginName) . '/Updates/*.php';
+                $files = _glob($pathToUpdates);
+                if (empty($files)) {
+                    $files = array();
+                }
+
+                foreach ($files as $file) {
+                    $fileVersion = basename($file, '.php');
+                    if (
+                        version_compare('3.13.0', $fileVersion) != 1
+                    ) {
+                        // since matomo 3.13.0 we basically don't want to see any plugin specific updates for core plugins
+                        // they should be instead in core/Updates/*
+                        $pluginsWithUnexpectedUpdates[$pluginName] = $file;
+
+                        var_export($pluginName . $file);
```

Comment:
Guess this debugging code could be removed.

Reply:
removed